sql/core/src/main/scala/org/apache/spark/sql/execution/debug/package.scala

```diff
@@ -105,13 +106,20 @@ package object debug {
    * @param plan the query plan for codegen
    * @return Sequence of WholeStageCodegen subtrees and corresponding codegen
    */
-  def codegenStringSeq(plan: SparkPlan): Seq[(String, String, ByteCodeStats)] = {
+  def codegenStringSeq(
+      plan: SparkPlan,
+      sparkSession: SparkSession): Seq[(String, String, ByteCodeStats)] = {
     val codegenSubtrees = new collection.mutable.HashSet[WholeStageCodegenExec]()
 
     def findSubtrees(plan: SparkPlan): Unit = {
       plan foreach {
         case s: WholeStageCodegenExec =>
           codegenSubtrees += s
+        case p: AdaptiveSparkPlanExec =>
+          // Find subtrees from original input plan of AQE.
+          val inputExecutedPlan = QueryExecution.prepareForExecution(
+            QueryExecution.preparations(sparkSession, None), p.inputPlan)
```

Comment:
Could we use `SparkSession.getActiveSession` to avoid the unnecessary changes?

Reply:
@maropu - good call, it's much cleaner. Updated.